### DISCUSSION TITLE
Alt key event

### DISCUSSION THREAD

**Discussion by wahur666** [tauri-apps/tauri#5325] [Q&A]
Hello, I'm using Preact + Phaser to create a game in Tauri. I setup a cursor grab and release event, based on blur/focus of the app, but there is some odd behavior, when I press ALT it will not trigger any more Canvas events, until it clicked back to the window. Is there an event, which I can listen for? I want to map the cursor release to this event.

**Comment by FabianLars** [comment]
Sounds like the menu takes focus on Alt press (i guess it can also happen if there's no menu :/ ) - i have no idea how to prevent that. Could you open a bug report (preferably with a minimal reproduction example, but not necessary) instead so that the team can take a closer look please? 🙏

**Reply by wahur666** [reply]
its not a big deal, but I will file a bug report shortly

**Comment by Arxoto** [comment]
Do you know how to solve this problem now? I had the same situation: window will lose focus when pressing the button 'ALT'.
I didn't find the bug report, or it's closed now?

**Reply by wahur666** [reply]
I mean, I found my answer, that replaced Tauri with Electron, then abandoned the whole project, but your mileage may vary

**Reply by wahur666** [reply]
https://github.com/tauri-apps/tauri/issues/5327

**Reply by Arxoto** [reply]
That's also a solution hhh. Anyway, thanks for your answer ~ : )
